arcgis_map_sdk_android/android/src/main/kotlin/dev/fluttercommunity/arcgis_map_sdk_android/ArcgisMapView.kt

```diff
@@ -263,6 +268,71 @@ internal class ArcgisMapView(
         }
     }
 
+    private fun onSetAutoPanMode(
+        call: MethodCall,
+        result: MethodChannel.Result
+    ) {
+        try {
+            val mode = call.arguments as String?
+            if (mode == null) {
+                result.error(
+                    "missing_data",
+                    "Invalid argument, expected an autoPanMode as string",
+                    null,
+                )
+                return
+            }
+            val autoPanMode = mode.autoPanModeFromString()
+            if (autoPanMode != null) {
+                mapView.locationDisplay.autoPanMode = autoPanMode
+                result.success(true)
+            } else {
+                result.error(
+                    "invalid_data",
+                    "Invalid argument, expected an AutoPanMode but got $mode",
+                    null,
+                )
+            }
+        } catch (e: Throwable) {
+            result.finishWithError(e, "Setting AutoPanMode failed.")
+        }
+    }
+
+    private fun onGetAutoPanMode(
+        call: MethodCall,
+        result: MethodChannel.Result
+    ) {
+        return result.success(mapView.locationDisplay.autoPanMode.name)
```

Comment:
I think arcgis is a java sdk and we might have a property that is not correctly annotated with `@Nullable`. In that case we would even get a null pointer.

I am almost sure that this is not the case, but lets double check 👍🏽 
You can also wrap this in a try catch just to be save.